sklearn/ensemble/tests/test_forest.py

```diff
@@ -1799,3 +1799,27 @@ def test_random_trees_embedding_feature_names_out():
         ]
     ]
     assert_array_equal(expected_names, names)
+
+
+@pytest.mark.parametrize("ForestClassifier", FOREST_CLASSIFIERS.values())
+@pytest.mark.parametrize(
```

Comment:
Is there a reason to pass these as parameters instead of having it in the function body? I don't know what the preferred style in sklearn is but I was naively expecting `X`, `y`, etc to be defined in the function body because they are always the same.